clients/curl-client.rst

```diff
@@ -15,7 +15,7 @@ To install the cURL client, run:
 Usage
 -----
 
-The cURL client needs a :ref:`message <message-factory>` and a :ref:`stream <message-factory>`
+The cURL client needs a :ref:`message <message-factory>` and a :ref:`stream <stream-factory>`
```

Comment:
This could be mentioned in the installation instructions as well. See guzzle5 doc.